input/capital-anonymized.md

```diff
@@ -0,0 +1,20 @@
+* The infrastructure that enables clients of the project to use the project's services.
+* A portal that allows clients and members of the public to see the accounting results and outstanding balances owed.
+* The programming language that the project is implemented in.
+* The infrastructure that runs tests and other code quality checks during development.
+* The platform publicly hosting the source code repository.
+* The version control system that records project development.
```

Comment:
I don't know that this materially contributed to the project. Including git would suggest including our text editors, the electricity company, our moms... 😝

Reply:
With our text editors and our moms we're individually indebted to them :) With electricity we've already paid for it. But with Git, it's the project that is indebted to it and no one has paid for it. Adding Git also didn't initially occur to me, but in a way, recognizing projects as valuable as Git that have zero market value is exactly the kind of thing that ABE is meant to address!